Castanet/Feed.php

```diff
@@ -166,6 +211,45 @@ public function setItunesBlock($itunes_block)
 	}
 
 	// }}}
+	// {{{ public function setAtomLink()
+
+	public function setAtomLink($atom_link)
+	{
+		$this->atom_link = strval($atom_link);
+	}
+
+	// }}}
+	// {{{ public function setItunesCategories()
+
+	public function setItunesCategories($itunes_category, array $itunes_subcategories = null)
```

Comment:
if you default to `array()` instead of `nul`l, you can remove the `is_null` check and just use `foreach`.